Move madrat to Suggests

johanneskoch94 opened the pull request:
- Move madrat to Suggests
- Bump version

johanneskoch94 committed:
Move madrat to Suggests

johanneskoch94 committed:
Bump version

0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q commented:
> ```
> Not all packages used in your code are listed in DESCRIPTION. The following are missing in file `/code/data-raw/create_internal_sources.R`: countrycode.
> ```

Wo er recht hat, hat er recht.

johanneskoch94 committed:
Add countrycode package to Suggests

johanneskoch94 commented:
Danke! (Wie hast du diese Warnung bekommen? Weder `devtools::check`, noch ` lucode::buildlibrary ` hat das bei mir geflagged...)

johanneskoch94 committed:
Fix bug

Fix bug where countries without PPP data were not getting the US conversion factors.

johanneskoch94 committed:
Bump version

johanneskoch94 committed:
Replace missing PPPs by MERs instead of with 1

johanneskoch94 committed:
Fill in using latest data first

0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q commented:
> Danke! (Wie hast du diese Warnung bekommen? Weder `devtools::check`, noch `lucode::buildlibrary` hat das bei mir geflagged...)

https://results.pre-commit.ci/run/github/351750647/1728652479.3pc77C1iSVKkOk7n67dGcg
